src/Messages/Interactions/TSMessage.m

```diff
@@ -132,6 +132,12 @@ - (nullable instancetype)initWithCoder:(NSCoder *)coder
         _attachmentIds = [NSMutableArray new];
     }
 
+    if (!_receivedAtDate) {
+        // TSIncomingMessage.receivedAt has been superceded by TSMessage.receivedAtDate.
```

Comment:
I know you prefer `readonly` properties, which make it hard for TSIncomingMessage to set this property.  Can I leave this?